core/capabilities/compute/compute.go

```diff
@@ -276,7 +276,7 @@ func (c *Compute) Close() error {
 	c.modules.close()
 	close(c.stopCh)
 	c.wg.Wait()
-	return nil
+	return c.registry.Remove(context.TODO(), CapabilityIDCompute)
```

Comment:
See my comment in the capabilities PR about doing this inside the standard capability job delegate, in one of the hooks.